test/check-application

```diff
@@ -78,8 +78,7 @@ class TestApplication(testlib.MachineCase):
         self.admin_s.execute("systemctl --user stop podman.service; systemctl --now --user enable podman.socket")
         self.restore_dir("/home/admin/.local/share/containers", reboot_safe=True)
         self.addCleanup(self.admin_s.execute, "systemctl --user stop podman.service podman.socket || true")
-        self.addCleanup(self.admin_s.execute, "podman rm --force --all || true")
-        self.addCleanup(self.admin_s.execute, "podman pod rm --force --all || true")
+        self.addCleanup(self.admin_s.execute, "podman system reset --force")
```

Comment:
But this also removes all images, and we want/need to keep them.

Reply:
.. I see, a subsequent commit then does that image restoration. This needs some squashing.